src/components/StorePage.vue

```diff
@@ -40,39 +39,9 @@
           <q-tab default slot="title" name="Products" label="Products" class="text-bold text-tertiary"/>
           <q-tab slot="title" name="Information" label="Information" class="text-bold text-tertiary"/>
           <!-- Targets -->
-          <q-tab-pane name="Products">
```

Comment:
might want to hold off on merging this in but i dont think it really matters. not done but it would have been broken either way with the recent platform updates